newdle/client/src/styles/forms.scss

```diff
@@ -0,0 +1,11 @@
+@import '../../styles/palette';
+
+%important-input {
+  width: 100%;
+  border-bottom: 1px solid $purple !important;
+  padding-bottom: 10px !important;
```

Comment:
I don't want to nitpick but I think it would be good to have a blank line between a CSS property and a selector.

Reply:
Sounds reasonale to me 😉